modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -855,6 +855,8 @@ class CV_EXPORTS DeviceInfo
 CV_EXPORTS void printCudaDeviceInfo(int device);
 CV_EXPORTS void printShortCudaDeviceInfo(int device);
 
+CV_EXPORTS void convertFp16Cuda(InputArray _src, OutputArray _dst, Stream& stream = Stream::Null());
```

Comment:
Thank you for the comment @jet47 .
Indeed, I do agree about removing ```Cuda``` prefix.

Actually I tried that, but then ```cv::convertFp16``` and ```cuda::convertFp16``` gets mixed up.
```
using namespace cv;
GpuMat src;
GpuMat dst;
convertFp16(src, dst); // is this cv::convertFp16 (CPU version) or cv::cuda::convertFp16 (GPU version)?
```
Now in this case, ```convertFp16``` could be the CPU version and the GPU version at the same time, so the compiler raised an error.
Another solution I thought was to change the input type from ```InputArray``` to ```GpuMat```, which still ends up in same situation.

Again, I do feel it's redundant to have ```Cuda``` prefix, so I'd like to fix this place, but unfortunately, this was the best I could reach (for now).